### DISCUSSION TITLE
Custom Scrapers

### DISCUSSION THREAD

**Discussion by metafates** [metafates/mangal#7] [Scrapers]
**Moved to [mangal-scrapers](https://github.com/metafates/mangal-scrapers) repository**

**Comment by OmiChrome** [comment]
hey , 

can u add soomething like search all scrapers at the same time . with tabs?

**Reply by OmiChrome** [reply]
or simply search all at once?

**Reply by metafates** [reply]
Yeah, I can make an option to select multiple scrapers to search with from here by pressing space (like you do with chapters). No tabs tho

<img width="357" alt="Screenshot 2022-09-01 at 15 49 01" src="https://user-images.githubusercontent.com/62389790/187917598-84791ade-eee5-46d6-a00b-c89cf590f973.png">

**Reply by metafates** [reply]
It works! #86 
Expect to see it in the next release

**Comment by OmiChrome** [comment]
oyeah thx :DD
